src/query/config/src/config.rs

```diff
@@ -2940,11 +2940,7 @@ pub struct DiskCacheConfig {
 #[serde(default, deny_unknown_fields)]
 pub struct SpillConfig {
     /// Path of spill to local disk. disable if it's empty.
-    #[clap(
-        long,
-        value_name = "VALUE",
-        default_value = "./.databend/temp/_query_spill"
-    )]
+    #[clap(long, value_name = "VALUE", default_value = "")]
```

Comment:
Should we change the `scripts/ci/deploy/config/databend-query-node-1.toml` together so that sqllogictest can cover the local spill?

Reply:
Yes, you can do it in another pr.